ts/screens/wallet/payment/TransactionSummaryScreen.tsx

```diff
@@ -319,14 +317,6 @@ class TransactionSummaryScreen extends React.Component<Props> {
                 <Text style={[styles.title, styles.lighterGray]}>
                   {I18n.t("wallet.firstTransactionSummary.updatedAmount")}
                 </Text>
-                {isAmountUpdated && (
```

Comment:
Since `isAmountUpdated` is always `true` (hard-coded value), what do you think to remove that flag and the relative conditional renders?

Reply:
I didn't notice it. Sure, let's do it.

Reply:
Fixed with cddb41b7536d0adbfd1b4e55da3b8b987c352e02.